src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        double area = Math.PI * Math.pow(radius, 2);
+        return Math.round(area);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle,"
+                + " area: " + calculateArea()
+                + " sq. units, radius: " + radius
+                + " units, color: " + getColor());
```

Comment:
The `draw` method should not print directly to the console. Consider using a more flexible approach, like returning the String representation of the object and letting the caller decide how to handle it.